helix-term/src/keymap.rs

```diff
@@ -200,17 +200,7 @@ impl Default for Keymaps {
             // extend_to_whole_line, crop_to_whole_line
 
 
-            key!('m') => Command::match_brackets,
-            // TODO: refactor into
-            // key!('m') => commands::select_to_matching,
-            // key!('M') => commands::back_select_to_matching,
-            // select mode extend equivalents
-
-            // key!('.') => commands::repeat_insert,
-            // repeat_select
-
-            // TODO: figure out what key to use
-            // key!('[') => Command::expand_selection, ??
+            key!('m') => Command::surround,
```

Comment:
`mm` is goto matching bracket and `ma` and `mi` might be added for around and inside objects, so we should call this something else. `match_mode` maybe ?

Reply:
`match_mode` sounds good. Still need to remap `ma` -> `ms` then we're good 👍🏻

Reply:
(The expand selection key that's commented out above should probably be `[o`/`]o` for expand/shrink. Shrink is unimplemented)

Reply:
Done